test/FastRouteRouterTest.php

```diff
@@ -650,4 +634,11 @@ public function testGenerateUriRaisesExceptionForMissingMandatoryParameters()
 
         $router->generateUri('foo');
     }
+
+    public function method() : Generator
```

Comment:
We don't need this method anymore, and also we can remove import Generator in top.